README.md

```diff
@@ -4,6 +4,8 @@ If you are unfamiliar with PerfView, there are [PerfView video tutorials](http:/
 As well as [Vance Morrison's blog](http://blogs.msdn.com/b/vancem/archive/tags/perfview) which also gives overview and getting 
 started information. 
 
+[![Build status](https://ci.appveyor.com/api/projects/status/fxtu3xa874whk2w0?svg=true)](https://ci.appveyor.com/project/sharwell/perfview)
```

Comment:
> Put a small description before this

I can, but note that it's not a practice I generally see. Examples:

* https://github.com/Microsoft/vs-threading
* https://github.com/dotnet/roslyn
* https://github.com/NuGet/NuGetGallery

> where the yml file is and logically what it is doing [etc.]

Considering the prevalence of AppVeyor for builds using Microsoft tool chains, I'm not sure this is necessary. While it could be added, my two concerns are the limited value it would provide and the possibility that the information would become out of date over time.

> and eventually tests

I am expecting tests to run with this pull request. After the webhooks are in place I can confirm this before the PR is merged.

> what happens with forks?

By default, nothing. Users can create their own AppVeyor accounts and connect them to their forks if they wish to do so.

> How does security work?

The build does not use any non-public information for any purpose, nor has any been exposed to it by this process. Security works by developers not putting confidential information into the repository itself. :smile: